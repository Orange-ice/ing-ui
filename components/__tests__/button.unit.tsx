import {Button} from '../index';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Button 组件测试', () => {
  it('快照检测', () => {
    const dom = renderer.create(<Button/>);
    expect(dom).toMatchSnapshot();
  });
});
