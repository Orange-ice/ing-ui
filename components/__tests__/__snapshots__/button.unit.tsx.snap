// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 组件测试 快照检测 1`] = `
<div>
  Button
</div>
`;
